src/vs/workbench/browser/layout.ts

```diff
@@ -66,6 +66,15 @@ export abstract class Layout extends Disposable implements IWorkbenchLayoutServi
 	private readonly _onZenMode: Emitter<boolean> = this._register(new Emitter<boolean>());
 	get onZenModeChange(): Event<boolean> { return this._onZenMode.event; }
 
+	private readonly _onFullscreen: Emitter<boolean> = this._register(new Emitter<boolean>());
```

Comment:
@sbatten the private members should have the exact same name as the public events starting with an underscore. E.g. _ `onCenteredLayoutChange` instead of `_onCenteredLayout`